tests/jballerina-integration-test/src/test/resources/configurables/encryptedSingleBalFile/encryptedConfig.bal

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/jballerina.java;
+import ballerina/'lang.config;
+
+configurable string password = ?;
+
+public function main() {
+    // Before encrypt
+    test:assertEquals("@encrypted:{ODYUoKSw0xW31eoxa/s2ESdBNgk1gX77txBIgpLC6NQ=}", password);
+     // After encrypt
+    test:assertEquals("waruna", config:decryptString(password));
+    print("Tests passed");
```

Comment:
we can remove this here also

Reply:
Resolving as per chat in configurable record PR